cli/dcoscli/config/main.py

```diff
@@ -109,6 +109,38 @@ def _unset(name):
 
     return 0
 
+def _format_config(file_value, effective_value, name=None, envvar_name=None,
+                   left_pad_fmt=None):
```

Comment:
lets remove `left_pad_fmt` from a param and create the format pattern in this function. since this function "formats" it makes sense to include it directly